src/api/app/views/webui2/shared/_requests_table.html.haml

```diff
@@ -0,0 +1,13 @@
+%table.requests-datatable.table.table-striped.table-bordered{ 'data-source': source_url,
```

Comment:
There are [responsive tables](https://getbootstrap.com/docs/4.1/content/tables/#responsive-tables) in bootstrap, there is also an [datatables extension](https://datatables.net/extensions/responsive/) to do this, with bootstrap integration. Check it out, or get a card in the backlog about this going please.